src/spyglass/utils/dj_helper_fn.py

```diff
@@ -8,6 +8,32 @@
 from .nwb_helper_fn import get_nwb_file
 
 
+def _delete(self: dj.Table, **kwargs) -> None:
```

Comment:
Maybe table instead of self? It is weird because it usually applies to a method on a class.